LandSandBoat/server
# Trust Gambit Condition IS_ECOSYSTEM, Rughadjeen, Algol

A grab bag of stuff!



**_I affirm:_**

## What does this pull request do?

Wires up Gambit Condition for IS_ECOSYSTEM
Adds Ancient Circle with Ecosystem Check to Shikaree Z
Wires up a good portion of Rughadjeen (including Ecosystem check for Holy Circle)
Added Trust Message Offsets to Rughadjeen's script.
And Algol additional effect while we're at it (see sql comments for Proc Rate Source)
The usual TODO comments where things are missing
Added Mobskill ID / Animation ID for Rughadjeen, Mihli, and Adelheid

## Steps to test these changes

Log in an appropriate GM Test character
Add Rughadjeen and Shikaree Z Trusts
Fight a Dragon! (`!fafnir`) - Shikaree Z now uses Ancient Circle
Fight something undead (`!zone 288` will take you right to undead) Rughadjeen will run through his gambits, and will use Holy Circle vs Undead Mobs
Add an Algol, equip it, hit stuff, observe added effect: fire on hits :fire: :fire: :fire:
